channels/iq.m3u

```diff
@@ -67,4 +67,21 @@ https://597f64b67707a.streamlock.net/anbsat.com/anb2/playlist.m3u8
 http://rtmp1.abnsat.com/hls/arabic.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Syriac" tvg-logo="" group-title="General",AshurSat
 https://5afd52b55ff79.streamlock.net/AshurSat/myStream/playlist.m3u8
-
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="" group-title="General",Al Fallujah
+http://api.new.livestream.com/accounts/4671269/events/4213685/broadcasts/198764660.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="" group-title="General",Al Fallujah
+https://api.new.livestream.com/accounts/4671269/events/4213685/broadcasts/198764660.secure.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="" group-title="General",Al Rafidain
+http://cdn.edge.tmed.pw/arrafidaintv/abr_live/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="" group-title="General",Al Masara Al Oula
+http://212.18.231.213:1935/almasar/_definst_/almasar_source/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="" group-title="General",Al Sumaria
+https://cllive.itworkscdn.net/alsumariatvlive/token=nva=1573435343~dirs=1~hash=0c67e20d42957215b17df/sumariamob.smil/playlist.m3u8
```

Comment:
This stream doesn't work for me. Tested with VLC Player and streamtest.in

Reply:
Nice catch. Placed wrong link - my apologies - new link should work.